packages/ember-blog/addon/components/pretty-collor.js

```diff
@@ -0,0 +1,4 @@
+import Component from '@glimmer/component';
+
+export default class PrettyColorComponent extends Component {
```

Comment:
minor, as this is glimmer component and component class has no behavior - we can drop this file